index.ts

```diff
@@ -27,9 +27,10 @@ function getStat(path: string) {
   try {
     return fs.statSync(path);
   } catch(error) {
-    if (error.code !== 'ENOENT') {
-      throw error;
+    if (error !== null && typeof error === 'object' && (error.code === 'ENOENT' || error.code === 'ENOTDIR')) {
```

Comment:
@lukemelia potential fix.

We don't currently warn on broken symlinks, as other dir executable libraries such as `tree`  do not either.

Reply:
@stefanpenner got it. This seems like a reasonable approach then.